dev-support/intellij/runConfigurations/Server1.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Server1" type="Application" factoryName="Application" nameIsGenerated="true">
+    <option name="MAIN_CLASS_NAME" value="org.apache.ratis.examples.debug.server.Server" />
+    <module name="ratis-examples" />
+    <option name="PROGRAM_PARAMETERS" value="127.0.0.1:10024" />
```

Comment:
Please see if we can easily generate this conf file from `ratis-examples/src/main/resources/conf.properties`.  If it is too complicated, please add a comment such as below.
```
    <!-- See ratis-examples/src/main/resources/conf.properties -->
```

Reply:
It doesn't seem to be easy, so I've added a comment